### ISSUE TITLE
Clicking on partially visible slide of centered carousel causes carousel to stop being centered

### ISSUE THREAD

**Issue by eugenefm** [FormidableLabs/nuka-carousel#679]
# Bugs and Questions

### Prerequisites

- [x] I've searched open [issues](https://github.com/FormidableLabs/nuka-carousel/issues) to make sure I'm not opening a duplicate issue
- [x] I have read through the docs before asking a question
- [x] I am using the latest version of nuka-carousel

### Describe Your Environment

- What version of nuka-carousel are you using? Current
- What browser are you using? Chrome v80
- What machine are you on? Mac

### Describe the Problem

If you have a carousel that is centered with partially visible slides on the sides. Clicking one of those partially visible slides will sroll it into view but the carousel will stop being centered. 

![Mar-18-2020 13-18-04](https://user-images.githubusercontent.com/35968807/76988879-d82bbd00-691b-11ea-81d5-a793c3978353.gif)

I would expect that this would either do nothing or scroll the slide into view in a centered position. 

Sandbox: https://codesandbox.io/s/frosty-bhaskara-vqi4y

**Comment by gregatarcurve** [comment]
I'm also running into this issue. The behavior appears to have been introduced in version 4.6.2. Forcing version 4.6.1 seems to fix the issue for me.

**Comment by nshikov** [comment]
it seems was introduced with this pr https://github.com/FormidableLabs/nuka-carousel/pull/639
with the slide `onClick` code and `handleSelfFocus ` stuff
```
  .slider-slide {
    outline: none;
    pointer-events: none;
  }

**Comment by sarmeyer** [comment]
This seems to be fixed with the most recent version 4.7.0 - I verified on the codesandbox that was linked and in the app demo
